STEERING-COMMITTEE.md

```diff
@@ -7,51 +7,55 @@ aliases:
   - /contributing/steering-committee/
 ---
 
-The Knative Steering Committee (KSC) is the ultimate authority for the Knative
-project, and governs all aspects of the project.
+The Knative Steering Committee (KSC) defines the governance for Knative
+in pursuit of the goal of an open, vibrant project with a productive
+community, a healthy ecosystem of vendors, and most of all satisfied
+customers running their workloads on Knative clusters.
 
 The governance of Knative is an open, living document, and will continue to
 evolve as the community and project change. We expect over time we will adapt
 the way we run this committee, based on feedback from the community.
 
+- [Mission](#mission)
 - [Charter](#charter)
 - [Delegated authority](#delegated-authority)
 - [Committee Meetings](#committee-meetings)
 - [Committee Mechanics](#committee-mechanics)
 - [Committee Members](#committee-members)
-  - [Allocation of seats](#allocation-of-seats)
+  - [Composition](#composition)
+  - [Nomination of representatives](#nomination-of-representatives)
 - [Decision process](#decision-process)
 - [Changes to the charter](#changes-to-the-charter)
 - [Getting in touch](#getting-in-touch)
 
+## Mission
+
+The Steering Committee's mission is to guide the project such that:
+
+1. Knative is a truly open effort which is inclusive and empowers contributors
+1. Knative core is clearly defined with a conformance suite
+1. Trademark usage is monitored and violations addressed
+
 ## Charter
 
-1. Define, evolve, and promote the vision, values, mission, and scope of the
-   project.
-1. Define and evolve project governance structures and policies, including
-   project roles and how collaborators become members, approvers, leads, and/or
-   administrators. This includes policy for the creation and administration of
-   [working groups](./working-groups/WORKING-GROUPS.md) and committees.
-1. Steward, control access, delegate access, and establishes processes
-   regarding, all Knative project resources and has the final say in the
-   disposition of those resources.
-1. Manage the Knative brand and decide which things can be called "Knative" and
-   how that mark can be used in relation to other efforts or vendors.
+1. Define, evolve, and promote the values of the project, and address any
+   [Code of Conduct](./CODE-OF-CONDUCT.md) issues that may arise, and
+   maintain confidentiality.
+1. Review and approve changes to the scope of Knative core.
```

Comment:
This omits a key point on which I understood last week's meeting to hinge:  Veto.  The veto ability was described last week as something required for Google as the trademark holder.  But is not described in this PR.

Changes approved by contributors and approved by TOC and approved by steering is the easy path.

The concerning path for the community is the veto path:  Changes approved by contributors and approved by TOC but vetoed by one entity for reasons perhaps unclear to the community.  This scenario is a problem for many of us and is enabled by this PR.  Scenarios where Google would use this implicit veto power are not described in the PR.

Without that description, this PR does not grant the Steering Committee the power described in this line of the PR.  It may appear so, but when viewed in the context of comments in the meeting it does not.  This PR would establish that Google approval is required for changes to the scope of Knative core.  Steering as a whole would only "review" in an advisory position, not "review and approve" as noted.  Google is the one and only necessary approver of scope changes in Knative core, regardless of non-Google steering member input.

There was some discussion last week around if or how similar veto could be granted to other companies.  But that's the opposite of what the community desires.  The community wants representation and doesn't want vetos.  Lacking that it's hard to know if your investment as an individual or corporate developer in the project is going to be arbitrarily blocked by Google.  Any of us are accustomed to having our work judged on its merits and having to refactor if a group of our peers or senior reviewers deem our work insufficient.  In open source we are not accustomed to our work having the extra hurdle of needing review/approval by one corporation.

While OSI's definition is specific to license, I think many of us here are looking for similar attributes to those described in the open source definition but with respect to project governance, eg:
 * No Discrimination Against Persons or Groups
 * No Discrimination Against Fields of Endeavor
 * Must Not Be Specific to a Product
 * Must Not Restrict Other Software
 * Must Be Technology-Neutral
As specified in this PR, Google optionally can do the opposite on any of those points to its unique advantage and the community has no recourse _within_ the project.